pkg/app/constants.go

```diff
@@ -1,3 +0,0 @@
-package app
```

Comment:
Why is this being removed?

Reply:
to avoid cyclic deps i was getting some time ago. I consolidated constants in one file.